### PR TITLE
Fix zone output list

### LINKED ISSUE #63: Outputs break after failed create
```
* module.gke-dev-cluster.local.cluster_type_output_zonal_zones: local.cluster_type_output_zonal_zones: concat: unexpected type list in list of type string in:

${concat(slice(var.zones,1,length(var.zones)), list(list()))}
```

### PR DESCRIPTION
**Pull request by morgante**
Fixes #63

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -50,12 +50,12 @@ locals {
     zonal    = "${var.region}"
   }
 
-  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list(list()))}"
-  cluster_type_output_zonal_zones    = "${concat(slice(var.zones,1,length(var.zones)), list(list()))}"
+  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list("")}"
```
Comment (Jberlinsky) [inline main.tf]: This is missing a trailing `)`, causing CI to fail

Comment (morgante) [reply main.tf]: Fixed

File: main.tf [hunk 2]
```diff
@@ -50,12 +50,12 @@ locals {
     zonal    = "${var.region}"
   }
 
-  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list(list()))}"
-  cluster_type_output_zonal_zones    = "${concat(slice(var.zones,1,length(var.zones)), list(list()))}"
+  cluster_type_output_regional_zones = "${concat(google_container_cluster.primary.*.additional_zones, list(""))}"
+  cluster_type_output_zonal_zones    = "${concat(slice(var.zones, 1, length(var.zones)), list(""))}"
 
   cluster_type_output_zones = {
     regional = "${local.cluster_type_output_regional_zones[0]}"
```
Comment (aaron-lane) [inline main.tf]: Should this also be changed to remove the index reference?
```suggestion
     regional = "${local.cluster_type_output_regional_zones}"
```

**Comment by aaron-lane** [comment]
Closed in favour of #132.

**Pull request closed without merging**